domains/lukass.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fraimerdelacruz",
+        "email": "fraimerdelacruzjaziel@gmail.com"
+    },
+    "record": {
+        "CNAME": "lukass"
```

Comment:
invalid CNAME record

Reply:
How can I edit?

Reply:
by editing the .json file in your fork (the .json file can be found in the domains folder)